src/auth/auth.controller.ts

```diff
@@ -22,8 +23,8 @@ export class AuthController {
   @UseGuards(LocalAuthGuard)
   @Post('login')
   @HttpCode(200)
-  async login(@Req() req: any): Promise<TokensDto> {
-    return req.user;
+  async login(@AuthorizedUser() tokens: TokensDto): Promise<TokensDto> {
+    return tokens;
   }
```

Comment:
I'd assume that `@AuthorizedUser` returns `User` type or something like that. Why does it return `TokensDto`? If we always return `TokensDto` maybe we should rename the decorator to `@AuthTokens()`?